src/module/__tests__/helpers-dice.test.ts

```diff
@@ -372,7 +477,6 @@ export default ({
       rollData.data.roll.dmg = [];
       await OseDice.sendAttackRoll(rollData);
       await waitForInput();
-      await waitForInput();
```

Comment:
Looking at the test, I'm not sure why I put multiple `waitForInput` here. I removed the multiples and tried running it, and it looks like the tests are just volatile due to timing, which makes sense. This is most likely why I put multiple `waitForInput` before, due to lag.

There might be an idea to rework the tests, to become more deterministic, especially since a lot of them waits for dice rolls, chat card rendering, and sheet rendering.

I'm not really sure what is the appropriate course of action here. I'll spend some time thinking about how to get to the problem.

Reply:
Hmmm, is it ok if I just merge and we'll make a ticket for deterministic waitforinput?

Reply:
I would be OK with that, it may be better to have the tests with the potential volatility compared to waiting until this is figured out.